guides/taxation/overview-of-taxation.md

```diff
@@ -0,0 +1,149 @@
+# Overview of taxation
+
+Solidus uses the `Spree::Adjustment` model to apply taxes to orders. This way,
+there can be multiple tax adjustments applied to each order.
+
+Using adjustments helps account for some of the complexities of tax, especially
+if a store sells internationally:
+
+- Orders may include products with different tax categories or rates.
+- Shipments may require special calculations if you are shipping to or from
+  locations where there are specific taxation rules for shipments.
+- Taxes may be included in a product's price or not in included in a product's
+  price depending on a country's taxation rules.
+
+<!-- TODO:
+  - Add links to the locations guide and specifically the [zones](#) article.
+  - Add links to the adjustments guide.
+-->
+
+Solidus uses the `Spree::TaxCategory` and `Spree::TaxRate` models to represent
```

Comment:
Nope. That's what the adjustments are for. 

The Tax Category and Tax Rate model specify the rules by which tax adjustments for order are created.